src/rules/image.ts

```diff
@@ -1,5 +1,12 @@
+import { isNullOrUndefined } from '../utils';
```

Comment:
I really wish you didn't modify the behavior of this rule in this PR.

Reply:
Well, I can just remove my modifications and it will still work. Should I?)